Cargo.toml

```diff
@@ -29,6 +29,7 @@ typescript = ["transforms", "swc_ecma_transforms_typescript"]
 utils = ["swc_ecma_utils"]
 view = ["dprint-swc-ext/view"]
 visit = ["swc_ecma_visit", "swc_visit", "swc_visit_macros", "swc_macros_common"]
+concurrent = ["swc_common/concurrent"]
```

Comment:
As mentioned in https://github.com/denoland/deno_ast/issues/219#issuecomment-2032383408 , I'm not sure this is a scenario we want to bother to support.

Reply:
unfortunately, a number of swc libraries bake-in the `swc_common`'s concurrent feature and including any of them prevents `deno_ast` from compiling:
- [`swc_error_reporters`](https://github.com/swc-project/swc/blob/7ac3b0a7dcee12a2aa977ae5bca18e38adaa5d18/crates/swc_error_reporters/Cargo.toml#L20-L22)
- [`swc_bundler`](https://github.com/swc-project/swc/blob/7ac3b0a7dcee12a2aa977ae5bca18e38adaa5d18/crates/swc_bundler/Cargo.toml#L66-L68)
- [`swc_ecma_lints`](https://github.com/swc-project/swc/blob/7ac3b0a7dcee12a2aa977ae5bca18e38adaa5d18/crates/swc_ecma_lints/Cargo.toml#L24-L26)
- [`swc_estree_compat`](https://github.com/swc-project/swc/blob/7ac3b0a7dcee12a2aa977ae5bca18e38adaa5d18/crates/swc_estree_compat/Cargo.toml#L27-L31)
- [`swc_node_bundler`](https://github.com/swc-project/swc/blob/7ac3b0a7dcee12a2aa977ae5bca18e38adaa5d18/crates/swc_node_bundler/Cargo.toml#L37-L39)
- [`swc_plugin_runner`](https://github.com/swc-project/swc/blob/7ac3b0a7dcee12a2aa977ae5bca18e38adaa5d18/crates/swc_plugin_runner/Cargo.toml#L62-L64)

i'm working from a fork of [farm](https://github.com/farm-fe/farm), which uses rayon for concurrent builds, and they [benchmark](https://github.com/farm-fe/performance-compare) against other bundlers using swc (e.g., rspack).

Reply:
Ah, I see. We can add a `concurrent` feature here, but no guarantee it's going to work in the future. It might be removed at any time due to the maintenance cost and you might have to contribute fixes to get it working again.